packages/conf-brotli/conf-brotli.0.0.1/opam

```diff
@@ -0,0 +1,13 @@
+opam-version: "1.2"
+maintainer: "fxfactorial"
+authors: "Edgar Aroutiounian"
+homepage: "https://github.com/google/brotli"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "BSD-3"
```

Comment:
According to [this](`https://github.com/google/brotli/blob/master/LICENSE`) libbrotli is `MIT` licensed.